src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowUpdate.cs

```diff
@@ -39,13 +39,79 @@ public RowUpdate(long rowId, ResultSet associatedResultSet, IEditTableMetadata a
         }
 
         /// <summary>
+        /// Sort order property. Sorts to same position as RowCreate
+        /// </summary>
+        protected override int SortId => 1;
```

Comment:
Is there a reason RowCreate and RowUpdate are both 1? Would there be any motivation to always do one before the other?